utils/build.ps1

```diff
@@ -2358,6 +2418,7 @@ if (-not $ToBatch) {
 }
 
 if (-not $SkipBuild) {
+  Invoke-BuildStep Build-SwiftTestingMacros $HostArch
```

Comment:
Can we move this next to the call to `Build-SwiftTesting` above?

Reply:
We can't move this right next to `Build-SwiftTesting` call because it  is for `$WindowsSDKArchs` not `$HostArch`. To what line you want to move this exactly?

Reply:
Wait, why is it being built for each architecture for the SDK? The macros run at compile time, so we should only have to build it for the host that the toolchain is being built for.

Reply:
Yes, TestingMacros is built for host only, Testing is built for each SDK. That's how I implemented. Am I missing anything?

Reply:
The macros should be running with the compiler (host) not the SDK environment (target). So, I'm not sure I understand why we are building it in all these cases.